packages/core/src/modules/proofs/__tests__/ProofRequest.test.ts

```diff
@@ -39,38 +40,34 @@ describe('ProofRequest', () => {
   })
 
   it('should throw an error if the proof request json contains an invalid structure', async () => {
-    const proofRequest = JsonTransformer.fromJSON(
-      {
-        name: 'ProofRequest',
-        version: '1.0',
-        nonce: '58d223e5-fc4d-4448-b74c-5eb11c6b558f',
-        requested_attributes: {
-          First: {
-            names: [],
-            restrictions: [
-              {
-                schema_id: 'q7ATwTYbQDgiigVijUAej:2:test:1.0',
-              },
-            ],
-          },
+    const proofRequest = {
+      name: 'ProofRequest',
+      version: '1.0',
+      nonce: '58d223e5-fc4d-4448-b74c-5eb11c6b558f',
+      requested_attributes: {
+        First: {
+          names: [],
+          restrictions: [
+            {
+              schema_id: 'q7ATwTYbQDgiigVijUAej:2:test:1.0',
+            },
+          ],
         },
-        requested_predicates: [
-          {
-            name: 'Timo',
-            p_type: '<=',
-            p_value: 10,
-            restrictions: [
-              {
-                schema_id: 'q7ATwTYbQDgiigVijUAej:2:test:1.0',
-              },
-            ],
-          },
-        ],
       },
-      ProofRequest
-    )
+      requested_predicates: [
+        {
+          name: 'Timo',
+          p_type: '<=',
+          p_value: 10,
+          restrictions: [
+            {
+              schema_id: 'q7ATwTYbQDgiigVijUAej:2:test:1.0',
+            },
+          ],
+        },
+      ],
+    }
 
-    // Expect 2 top level validation errors
-    expect(MessageValidator.validate(proofRequest)).rejects.toHaveLength(2)
+    expect(() => JsonTransformer.fromJSON(proofRequest, ProofRequest)).toThrowError(ClassValidationError)
```

Comment:
losing that we have two validation errors (and just check if it is a class validatino error)